test/src/test/resources/logback-test.xml

```diff
@@ -78,7 +78,7 @@
 
     <root level="INFO">
         <!--<appender-ref ref="FILE" />-->
-        <!--<appender-ref ref="STDOUT" />-->
+        <appender-ref ref="STDOUT" />
```

Comment:
Can you please remove these changes to the logback since it would create a lot of noise in the GitHub action verifications